packs/datadog/actions/run.py

```diff
@@ -0,0 +1,14 @@
+import importlib
+from st2actions.runners.pythonrunner import Action
+
+
+class ActionWrapper(Action):
+    def __init__(self, config):
+        self.config = config
+
+    def run(self, **kwargs):
```

Comment:
Yes, I started to write everything by hand but it took me a lot of time so I tried to auto-generate the biggest part but I had to write some things by hand. (I.e the functions taking an ID for editing or deleting things but that's not very hard, the type of the argument in the action file etc)

The main reason that I have not declared the supported arguments is because some arguments may be optional. If I set a default value, it will alter the behavior of the function. But I think I should declare only the required arguments and then the rest will be in the kwargs.

By the way, this method run is only a wrapper, so we need to keep the **kwargs here.

I'll still work on the pack for improving it, including unit tests too.